README.md

```diff
@@ -2,7 +2,11 @@
 Embedding a video in a webpage
 
 BUILT WITH
+-HTML
 
-HTML
+-CSS
```

Comment:
If you want to make a list in markdown, there should be a space between the `-` and the `list`. See:

```md
<!-- use this -->

- HTML
- CSS

<!--- not this --->

-HTML
-CSS
```
Kindly check [this link for more details](https://guides.github.com/features/mastering-markdown/):